gips/test/sys/t_modis.py

```diff
@@ -28,8 +28,15 @@ def setup_modis_data(pytestconfig):
     outcome = envoy.run(cmd_str)
     logger.info("MODIS data download complete.")
     if outcome.status_code != 0:
-        raise RuntimeError("MODIS data setup via `gips_inventory` failed",
-                           outcome.std_out, outcome.std_err, outcome)
+        msg = ("MODIS data setup via `gips_inventory` technically failed, but this may be due to false"
```

Comment:
Ok...after discussion with @ags-tolson this is all pre-#201, so that's not what make the non-zero exit status.  @ags-tolson will take a minute this afternoon after the modis maintenance window and see why the exit status is non zero.